.npmignore

```diff
@@ -24,9 +23,6 @@ __pycache__
 *.py
 ipywidgets/widgets
 ipywidgets/tests
-.bowerrc
 .travis.yml
-bower.json
-build_css.js
```

Comment:
except for build_css which is completely removed since we don't build css anymore.